CHANGELOG.md

```diff
@@ -1,13 +1,39 @@
 # Change Log
 
-## [1.7.0](https://github.com/chef/chef-provisioning-aws/tree/1.7.0) (2015-11-19)
-[Full Changelog](https://github.com/chef/chef-provisioning-aws/compare/v1.6.1...1.7.0)
+## [Unreleased](https://github.com/chef/chef-provisioning-aws/tree/HEAD)
```

Comment:
Check out https://github.com/chef/chef-provisioning/blob/master/CONTRIBUTING.md#release-process (specifically \#5) - if you specify `--future-release 1.8.0` it will put that here